circe.el

```diff
@@ -442,7 +442,7 @@ See `circe-channels' for more details."
 
 (defgroup circe-format nil
   "Format strings for Circe.
-All these formats always allow the {mynick} and {chattarget} format
+All these formats (except for `circe-format-buffer-name') always allow the {mynick} and {chattarget} format
```

Comment:
I think it's a bit more complicated than that. `mynick` and `chattarget` are available for format strings used by `circe-display`. For example `circe--server-generate-buffer` is affected as well.

Reply:
I've removed my change of the comment for now, not sure how to best address this.